pyrsa/data/dataset.py

```diff
@@ -49,6 +50,35 @@ def __init__(self, measurements=None, descriptors=None,
         self.obs_descriptors = obs_descriptors
         self.channel_descriptors = channel_descriptors
 
+    def __repr__(self):
+        """
+        defines string which is printed for the object
+        """
+        return (f'pyrsa.data.{self.__class__.__name__}(\n'
+                f'measurements = \n{self.measurements}\n'
+                f'descriptors = \n{self.descriptors}\n'
+                f'obs_descriptors = \n{self.obs_descriptors}\n'
+                f'channel_descriptors = \n{self.channel_descriptors}\n'
+                )
+
```

Comment:
This works, another approach would be a triple-quoted f string:

```
f"""
pyrsa.data.{self.__class__.__name__}(
    measurements = {self.measurements}
    descriptors = {self.descriptors}
    obs_descriptors = {self.obs_descriptors}
    channel_descriptors = {self.channel_descriptors}
)
"""
```

Reply:
True, in this case I like the one string per output version. Do you have a strong preference for the triple-quoted string?

Reply:
nope, no preference